doc/nom-guide/src/chapter_2.md

```diff
@@ -0,0 +1,105 @@
+# Chapter 2: Tags and Character Classes
+
+The simplest _useful_ regex you can write is one which
+has no special characters, it just matches a string.
+
+Imagine, for example, the regex `/abc/`. It simply matches when the string
+`"abc"` occurs.
+
+In `nom`, we call a simple collection of bytes a tag. Because
+these are so common, there already exists a function called `tag()`.
+This function returns a parser for a given string.
+
+<div class="example-wrap" style="display:inline-block"><pre class="compile_fail" style="white-space:normal;font:inherit;">
+ **Warning**: `nom` has multiple different definitions of `tag`, make sure you use this one for the
```

Comment:
that will need a link to the future chapter about parsers of complete VS streaming inputs

Reply:
Agreed; I'll add it in when that chapter is written.